Python/bytecodes.c

```diff
@@ -0,0 +1,3983 @@
+            opcode = _PyOpcode_Deopt[_Py_OPCODE(*next_instr)];
+            PRE_DISPATCH_GOTO();
+            // CPython hasn't traced the following instruction historically
+            // (DO_TRACING would clobber our extended oparg anyways), so just
+            // skip our usual cframe.use_tracing check before dispatch. Also,
+            // make sure the next instruction isn't a RESUME, since that needs
+            // to trace properly (and shouldn't have an extended arg anyways):
+            assert(opcode != RESUME);
+            DISPATCH_GOTO();
+        }
+
+        // stack effect: ( -- )
+        inst(EXTENDED_ARG_QUICK) {
+            assert(cframe.use_tracing == 0);
+            assert(oparg);
+            int oldoparg = oparg;
+            NEXTOPARG();
+            oparg |= oldoparg << 8;
+            DISPATCH_GOTO();
+        }
+
+        // stack effect: ( -- )
+        inst(CACHE) {
+            Py_UNREACHABLE();
+        }
+
+
+// END BYTECODES //
+
+    }
+ handle_eval_breaker:;
+ unbound_local_error:;
+ error:;
+}
+
+// Families go below this point //
+
+family(binary_op) =
+    BINARY_OP, BINARY_OP_ADAPTIVE, BINARY_OP_ADD_FLOAT,
+    BINARY_OP_ADD_INT, BINARY_OP_ADD_UNICODE, BINARY_OP_INPLACE_ADD_UNICODE,
+    BINARY_OP_MULTIPLY_FLOAT, BINARY_OP_MULTIPLY_INT, BINARY_OP_SUBTRACT_FLOAT;
+    BINARY_OP_SUBTRACT_INT;
+family(binary_subscr) =
+    BINARY_SUBSCR, BINARY_SUBSCR_ADAPTIVE, BINARY_SUBSCR_DICT,
+    BINARY_SUBSCR_GETITEM, BINARY_SUBSCR_LIST_INT, BINARY_SUBSCR_TUPLE_INT;
+family(call) =
+    CALL, CALL_ADAPTIVE, CALL_PY_EXACT_ARGS,
+    CALL_PY_WITH_DEFAULTS, CALL_BOUND_METHOD_EXACT_ARGS, CALL_BUILTIN_CLASS,
+    CALL_BUILTIN_FAST_WITH_KEYWORDS, CALL_METHOD_DESCRIPTOR_FAST_WITH_KEYWORDS, CALL_NO_KW_BUILTIN_FAST,
+    CALL_NO_KW_BUILTIN_O, CALL_NO_KW_ISINSTANCE, CALL_NO_KW_LEN,
+    CALL_NO_KW_LIST_APPEND, CALL_NO_KW_METHOD_DESCRIPTOR_FAST, CALL_NO_KW_METHOD_DESCRIPTOR_NOARGS,
+    CALL_NO_KW_METHOD_DESCRIPTOR_O, CALL_NO_KW_STR_1, CALL_NO_KW_TUPLE_1;
+    CALL_NO_KW_TYPE_1;
+family(compare_op) =
+    COMPARE_OP, COMPARE_OP_ADAPTIVE, COMPARE_OP_FLOAT_JUMP,
+    COMPARE_OP_INT_JUMP, COMPARE_OP_STR_JUMP;
+family(extended_arg) = EXTENDED_ARG, EXTENDED_ARG_QUICK;
+family(for_iter) =
+    FOR_ITER, FOR_ITER_ADAPTIVE, FOR_ITER_LIST;
+    FOR_ITER_RANGE;
+family(jump_backward) = JUMP_BACKWARD, JUMP_BACKWARD_QUICK;
+family(load_attr) =
+    LOAD_ATTR, LOAD_ATTR_ADAPTIVE, LOAD_ATTR_CLASS,
+    LOAD_ATTR_GETATTRIBUTE_OVERRIDDEN, LOAD_ATTR_INSTANCE_VALUE, LOAD_ATTR_MODULE,
+    LOAD_ATTR_PROPERTY, LOAD_ATTR_SLOT, LOAD_ATTR_WITH_HINT,
+    LOAD_ATTR_METHOD_LAZY_DICT, LOAD_ATTR_METHOD_NO_DICT, LOAD_ATTR_METHOD_WITH_DICT;
+    LOAD_ATTR_METHOD_WITH_VALUES;
+family(load_const) = LOAD_CONST, LOAD_CONST__LOAD_FAST;
+family(load_fast) = LOAD_FAST, LOAD_FAST__LOAD_CONST, LOAD_FAST__LOAD_FAST;
+family(load_global) =
+    LOAD_GLOBAL, LOAD_GLOBAL_ADAPTIVE, LOAD_GLOBAL_BUILTIN;
+    LOAD_GLOBAL_MODULE;
+family(resume) = RESUME, RESUME_QUICK;
```

Comment:
This family will be removed once you resolve this morning's merge conflict (these opcodes were re-combined).